hybridse/src/udf/default_udf_library.cc

```diff
@@ -666,6 +666,35 @@ void DefaultUdfLibrary::InitStringUdf() {
                 select date_format(date(1590115420000),"%Y-%m-%d");
                 --output "2020-05-22"
             @endcode)");
+    RegisterExternal("like_match")
+        .args<StringRef, StringRef, StringRef>(reinterpret_cast<void *>(
+            static_cast<void (*)(codec::StringRef *, codec::StringRef *, codec::StringRef *, bool *, bool *)>(
+                udf::v1::like_match)))
+        .return_by_arg(true)
+        .returns<Nullable<bool>>()
+        .doc(R"r(
+                @brief perform SQL LIKE match
```

Comment:
Please add enough rules and examples for "like_match" in `.doc()`, since it is really important and there are no standard rules for that.